.github/CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# ref: https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
+
+pdp/ @bdewilde @kaylawilding @Eyong20
+zogotech/ @anzhely
```

Comment:
it looks like code owners is case sensitive so either change this to `ZogoTech/` or change [this folder](https://github.com/datakind/student-success-tool/tree/develop/synthetic-data/ZogoTech) to "zogotech" (latter preferred?

Reply:
Yeah, good catch. I'll rename that directory here in this PR once I'm on my non-work machine.